### ISSUE TITLE
ImportError: /lib/python3.10/site-packages/lxml/etree.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send

### ISSUE THREAD

**Issue by taylorreiter** [dib-lab/genome-grist#120]
I abstracted the rules from genome-grist, but receive this error message both inside and outside the genome grist pipeline when using version 0.7.3
```
[Tue Dec  7 18:29:26 2021]
rule make_genome_info_csv:
    output: outputs/genbank_genomes/GCF_002865385.1.info.csv
    jobid: 78
    wildcards: acc=GCF_002865385.1
    resources: tmpdir=/tmp, mem_mb=8000

Activating conda environment: /home/tereiter/github/2021-metapangenome-example/.snakemake/conda/86762d5f0fa5b4ff96cae083277aef7f
Traceback (most recent call last):
  File "/home/tereiter/github/2021-metapangenome-example/scripts/genbank_genomes.py", line 11, in <module>
    from lxml import etree
ImportError: /home/tereiter/github/2021-metapangenome-example/.snakemake/conda/86762d5f0fa5b4ff96cae083277aef7f/lib/python3.10/site-packages/lxml/etree.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

**Comment by taylorreiter** [comment]
Perhaps relevant:
```
# Name                    Version                   Build  Channel
lxml                      4.6.1                    pypi_0    pypi
```

**Comment by taylorreiter** [comment]
I made a new conda environment for this step and installed `lxml=4.6.4` from conda, and the error went away.

**Comment by ctb** [comment]
thanks, Taylor. I'll review your PR in a bit, unless it's urgent for you, but I wanted to note something that caught my attention - it looks like the root of this problem may lie in the installation of python 3.10, which is being used in the conda environment; see string `lib/python3.10/site-packages/lxml/etree.cpython-310-x86_64-linux-gnu.so` above, showing that it's in a python3.10 lib dir!

This might be why it's not caught by tests, and it's something to keep in mind for the future.

**Comment by taylorreiter** [comment]
Ah nice catch! Not urgent for me at all.

**Comment by ctb** [comment]
Fixed by #121 / #122.
